### ISSUE TITLE
Silver white character to scales on skin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9966]
**Preferred term label:**
Silver white character to scales on skin

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tinea pedis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
This term is being used in a different context:
[White scaling skin HP:0040190](https://hpo.jax.org/browse/term/HP:0040190)
but is close. HPO does not like to link terms to specific diseases (tinea). Perhaps this is sufficient? If not, what is the differentia?

**Comment by MickeySegal** [comment]
That is good.  OK to close.

**Comment by pnrobinson** [comment]
OK!
